File: javascript/properties-methods-and-instances/classBook.js
class Book {
  constructor(titulo, autor, genero, numPaginas) {
  // Attributes
    this.titulo = titulo;
    this.autor = autor;
    this.genero = genero;
    this.numPaginas = numPaginas;
  }

  // Methods
  mostrarDetalles() {
    console.log(`Titulo: ${this.titulo}, \nAutor: ${this.autor}, \nGenero: ${this.genero}, \nPaginas: ${this.numPaginas}`);
  }
}

// This is the way to EXPORT a class in Node.js
module.exports = Book;

File: python/properties-methods-and-instances/exercise01/classBook.py
class Book:
  def __init__(self, titulo, autor, genero, numPaginas):
  # Attributes
    self.titulo = titulo
    self.autor = autor
    self.genero = genero
    self.numPaginas = numPaginas

  # Method to show the details of the book
  def mostrarDetalles(self):
    print(f"1. Titulo: {self.titulo} \n2. Autor: {self.autor} \n3. Genero: {self.genero} \n4. Paginas: {self.numPaginas}")